lute/main.py

```diff
@@ -108,12 +111,34 @@ def _start(args):
         "--config",
         help="Path to override config file.  Uses lute/config/config.yml if not set.",
     )
+    parser.add_argument(
+        "--upgrade",
+        action="store_true",
+        help="Upgrade to latest version (doesn't work with docker)",
```

Comment:
Not exactly accurate: this shouldn't be used with Docker, the image should be rebuilt.